public/nginx.conf

```diff
@@ -14,6 +14,79 @@ events {
 }
 
 
+##########################CSP VARIABLES#######################
+# CSP Settings based on:
+# LogRocket - https://docs.logrocket.com/docs/troubleshooting-sessions#due-to-content-security-policy
+# GTM - https://developers.google.com/tag-platform/security/guides/csp
+# Stripe - https://docs.stripe.com/security/guide?csp=csp-connect#content-security-policy
+
+########################## script src #######################
+set $SCRIPT "script-src 'self' 'unsafe-eval'";
+set $SCRIPT "${SCRIPT} https://docs.estuary.dev ";
+set $SCRIPT "${SCRIPT} https://cdn.logrocket.io";
+set $SCRIPT "${SCRIPT} https://cdn.lr-ingest.io";
+set $SCRIPT "${SCRIPT} https://cdn.lr-in.com";
+set $SCRIPT "${SCRIPT} https://cdn.lr-in-prod.com";
+set $SCRIPT "${SCRIPT} https://cdn.lr-ingest.com";
+set $SCRIPT "${SCRIPT} https://cdn.ingest-lr.com";
+set $SCRIPT "${SCRIPT} https://cdn.lr-intake.com";
+set $SCRIPT "${SCRIPT} https://cdn.intake-lr.com";
+set $SCRIPT "${SCRIPT} https://*.google-analytics.com";
+set $SCRIPT "${SCRIPT} https://*.analytics.google.com";
+set $SCRIPT "${SCRIPT} https://*.googletagmanager.com";
+set $SCRIPT "${SCRIPT} https://*.g.doubleclick.net";
+set $SCRIPT "${SCRIPT} https://*.google.com";
+set $SCRIPT "${SCRIPT} https://*.reo.dev";
+set $SCRIPT "${SCRIPT} https://*.stripe.network";
+set $SCRIPT "${SCRIPT} https://*.js.stripe.com";
+set $SCRIPT "${SCRIPT} https://js.stripe.com";
+set $SCRIPT "${SCRIPT} https://maps.googleapis.com";
```

Comment:
This is now locked down. Still have `eval` as that is how Google Tag Manager runs.